src/scripts/main.js

```diff
@@ -2,7 +2,41 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tableElement = document.querySelector('.dashboard');
 
-// write your code here
+const generateTableData = (data, row) => {
+  for (const value of Object.values(data)) {
+    const tableData = document.createElement('td');
+
+    tableData.textContent = value;
+
+    row.appendChild(tableData);
+  }
+};
+
+const generateTableRows = (table, data) => {
+  for (const person of data) {
+    const tableRow = document.createElement('tr');
+
+    const { name: fullName, sex, born, died } = person;
+
+    const age = died - born;
+    const century = Math.ceil(died / 100);
+    const gender = sex === 'm' ? 'Male' : 'Female';
+
+    const personData = {
+      fullName,
+      gender,
+      born,
+      died,
+      age,
+      century,
+    };
+
+    generateTableData(personData, tableRow);
```

Comment:
it will be great to return `tableRow` element from the function and rename it to `createTableRow`